draft-ietf-dnsop-generalized-notify.md

```diff
@@ -470,29 +461,64 @@ Alternative solutions (such as signing notifications and validating
 their signatures) appear significantly more expensive without tangible
 benefit.
 
+The specification does not require DSYNC records to be signed with
+DNSSEC, as to not exclude schemes that are otherwise authenticated (such
+as via TSIG). The effects from spoofing the response to a DSYNC query
```

Comment:
I think SIG(0) is a better example than TSIG here, as the mere concept of setting up shared TSIG keys between child and parent would make anyones head hurt. But it is a detail, not that important.

Reply:
Ack (and I actually meant SIG(0) and mistyped)

Reply:
done